packages/web-react/src/components/Toast/demo/ToastStaticToast.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from '../../Button';
+import Toast from '../Toast';
+import ToastBar from '../ToastBar';
+
+const ToastStaticToast = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const buttonLabel = isOpen ? 'Hide' : 'Show';
+
+  // Experimental, Chrome 94+ on Android only:
+  // Enable CSS to detect the presence of virtual keyboard.
+  if ('virtualKeyboard' in navigator) {
+    // @ts-expect-error 'navigator.virtualKeyboard' is of type 'unknown'.
+    navigator.virtualKeyboard.overlaysContent = true;
+  }
```

Comment:
This actually belongs to the `ToastDynamicToastQueue` demo:

<img width="338" alt="obrazek" src="https://github.com/lmc-eu/spirit-design-system/assets/5614085/176cc5e3-27a4-42de-ba07-2483cbf97b56">

It't not about the dynamic queue, but it's placed there 🤷🏻‍♂️.

Reply:
i forgot it there - removed, thank you